src/uu/chroot/src/error.rs

```diff
@@ -4,94 +4,66 @@
 // file that was distributed with this source code.
 // spell-checker:ignore NEWROOT Userspec userspec
 //! Errors returned by chroot.
-use std::collections::HashMap;
 use std::io::Error;
 use thiserror::Error;
 use uucore::display::Quotable;
 use uucore::error::UError;
 use uucore::libc;
-use uucore::locale::{get_message, get_message_with_args};
+use uucore::translate;
 
 /// Errors that can happen while executing chroot.
 #[derive(Debug, Error)]
 pub enum ChrootError {
     /// Failed to enter the specified directory.
-    #[error("{}", get_message_with_args("chroot-error-cannot-enter", HashMap::from([
-        ("dir".to_string(), _0.quote().to_string()),
-        ("err".to_string(), _1.to_string())
-    ])))]
+    #[error("{}", translate!("chroot-error-cannot-enter", "dir" => _0.quote(), "err" => _1))]
     CannotEnter(String, #[source] Error),
 
     /// Failed to execute the specified command.
-    #[error("{}", get_message_with_args("chroot-error-command-failed", HashMap::from([
-        ("cmd".to_string(), _0.to_string().quote().to_string()),
-        ("err".to_string(), _1.to_string())
-    ])))]
+    #[error("{}", translate!("chroot-error-command-failed", "cmd" => _0.to_string().quote(), "err" => _1))]
     CommandFailed(String, #[source] Error),
 
     /// Failed to find the specified command.
-    #[error("{}", get_message_with_args("chroot-error-command-not-found", HashMap::from([
-        ("cmd".to_string(), _0.to_string().quote().to_string()),
-        ("err".to_string(), _1.to_string())
-    ])))]
+    #[error("{}", translate!("chroot-error-command-not-found", "cmd" => _0.to_string().quote(), "err" => _1))]
```

Comment:
I think calling `to_string()` before calling `quote()` is not necessary as it is already a `String`.